src/server/api/routes/routes.router.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+const express = require("express");
+const router = express.Router({
+  mergeParams: true
+});
+
+const routesController = require("../controllers/routes.controller");
+
+router.post("/", (req, res, next) => {
+  routesController
+    .createArouteWithWaypoints({
+      body: req.body
+    })
+    .then(result => res.json(result))
+    .catch(next);
+});
```

Comment:
We renamed `routes` to `suggested-routes` all around. You need to move this to this file: https://github.com/HackYourFuture-CPH/ov-class09-fp/blob/develop/src/server/api/routes/suggested-routes.router.js